src/node/setupServer.ts

```diff
@@ -7,18 +7,23 @@ import { RequestHandler, MockedRequest } from '../handlers/requestHandler'
 import { getResponse } from '../utils/getResponse'
 import { parseRequestBody } from '../utils/parseRequestBody'
 
+type RequestHandlersList = RequestHandler<any, any>[]
```

Comment:
Similar type definition is present in `setupWorker`. Consider moving to `glossary` and reusing.